privacy-protections/storage-partitioning/serviceworker.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Portions of the code derived from privacytests.org source code (https://github.com/arthuredelstein/privacytests.org)
+ * Copyright 2018 Arthur Edelstein
+ * MIT License (https://mit-license.org/)
+ */
+
+let data; // undefined
+
+self.addEventListener('install', () => {
+    self.skipWaiting();
+});
+
+self.addEventListener('activate', function (event) {
+    event.waitUntil(self.clients.claim());
+});
+
+self.addEventListener('fetch', async (event) => {
+    const scope = self.registration.scope;
+    const shortPath = event.request.url.split(scope)[1];
```

Comment:
nit: I'd put `new URL(event.request.url)` into a variable and reused it here (url.pathname) and below.